packages/console/src/pages/Connectors/components/CreateForm/index.tsx

```diff
@@ -97,14 +117,40 @@ const CreateForm = ({ onClose, isOpen: isFormOpen, type }: Props) => {
                 <div className={styles.name}>
                   <UnnamedTrans resource={name} />
                 </div>
-                <div className={styles.connectorId}>{id}</div>
+                {type !== ConnectorType.Social && <div className={styles.connectorId}>{id}</div>}
                 <div className={styles.description}>
                   <UnnamedTrans resource={description} />
                 </div>
               </Radio>
             ))}
           </RadioGroup>
         )}
+        {activeGroup && activeGroup.connectors.length > 1 && (
+          <div className={styles.platforms}>
+            <div className={styles.title}>
+              <UnnamedTrans resource={activeGroup.name} />
+              {t('connectors.add_multi_platform')}
+            </div>
+            <RadioGroup
+              type="plain"
+              name="connector"
+              value={activeConnectorId}
+              onChange={setActiveConnectorId}
+            >
+              {activeGroup.connectors.map(
+                ({ platform, id, enabled }) =>
+                  platform && (
+                    <Radio
+                      key={id}
+                      value={id}
+                      title={connectorPlatformLabel[platform]}
+                      isDisabled={enabled}
+                    />
+                  )
+              )}
+            </RadioGroup>
+          </div>
+        )}
```

Comment:
nit: maybe, it's a good place to extract them as an independent component? So  outer scope does not care about the connectorId change.

Reply:
updated